docs/developer/tutorials/snmp/profile-format.md

```diff
@@ -470,6 +470,40 @@ This example will submit two metrics `snmp.upsBasicStateOutputState.OnLine` and
 
 [Example of flag_stream usage in a profile](https://github.com/DataDog/integrations-core/blob/e64e2d18529c6c106f02435c5fdf2621667c16ad/snmp/datadog_checks/snmp/data/profiles/apc_ups.yaml#L60-L127).
 
+#### Extract value
+
+If the metric value to be submitted is from a OID with string value and needs to be extracted from it, you can use extract value feature.
```

Comment:
I would add that extract_value accepts a valid regular expression with the metric part of the string to be defined in the first group. Possibly adding some examples:

```
- stripping the C unit from a temperature value: '(\d+)C' 
- stripping the USD unit from a currency value: 'USD(\d+)'
- stripping the F unit from a temperature value with spaces between the metric and the unit: '(\d+)\ *C' 
```

Maybe also a link to regex online helper like https://regexr.com/

Reply:
@pducolin Thx for the suggestion. Improved the doc here https://github.com/DataDog/integrations-core/pull/8622/commits/26858354550128fbbda58103dc4730fa1563d910